rtv/subscription_page.py

```diff
@@ -73,6 +106,42 @@ def close_subscriptions(self):
 
         self.active = False
 
+    @SubscriptionController.register(Command('DELETE'))
+    def delete_reddit(self):
+        "Delete the selected reddit"
+
+        context = self.content.name
+        data = self.get_selected_item()
+        listing = data['object']
+        name = data['name']
+
+        if isinstance(listing, praw.objects.Multireddit) and \
+                self.term.prompt_y_or_n('Delete {}? (y/n): '.format(name)):
+            with self.term.loader('Deleting {} from {}'.format(name, context)):
+                self.reddit.delete_multireddit(listing.name)
+                self.refresh_content()
+        else:
+            with self.term.loader('Deleting {} from {}'.format(name, context)):
+                if hasattr(self.content, '_multireddit'):
+                    self.content._multireddit.add_subreddit(
+                                            listing.display_name, _delete=True)
+                elif isinstance(listing, praw.objects.Subreddit):
+                    listing.unsubscribe()
+                self.refresh_content()
+
+    @SubscriptionController.register(Command('SUBMISSION_TOGGLE_COMMENT'))
```

Comment:
In this case, I think we should add a new binding since ``SUBMISSION_*`` is reserved for the submission page. It can still be bound to the space bar (``0x20``) but it should have a separate definition like ``SUBSCRIPTION_VIEW_MULTIREDDIT``.